### PR TITLE
Add custom mobilenet wrapper

### PR DESCRIPTION
**Pull request by samwinebrake**
Currently, behavioral benchmarks are not working on these mobilenets due to an assertion error related to number of logits results. This wrapper slices the background class out which is not typical in pytorch models.

### CODE REVIEW

**Comment by samwinebrake** [comment]
All tests pass, unit plugin test parsing wrong

**Review by mike-ferguson — APPROVED** [review]
